website/integrations/services/grafana/index.mdx

```diff
@@ -56,16 +47,6 @@ data "authentik_scope_mapping" "scope-openid" {
   name = "authentik default OAuth Mapping: OpenID 'openid'"
 }
 
-resource "authentik_scope_mapping" "scope-grafana-roles" {
```

Comment:
This should also be removed below in the `property_mappings` list.

Reply:
I think you clicked the wrong line, but I think I understood what you meant, line 86.